alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/validation/FieldValidationUtility.java

```diff
@@ -33,14 +33,13 @@
 
 import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.errors.AlertFieldStatus;
-import com.synopsys.integration.alert.common.descriptor.config.field.validators.ValidationResult;
 import com.synopsys.integration.alert.common.enumeration.FieldType;
 import com.synopsys.integration.alert.common.rest.model.FieldModel;
 import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
 
 @Component
-public class FieldValidationAction {
-    private final Logger logger = LoggerFactory.getLogger(FieldValidationAction.class);
+public class FieldValidationUtility {
```

Comment:
This file contains some secondary changes. It is primarily making more methods public and updating the return types instead of modifying input parameters.